src/json_file_module.F90

```diff
@@ -1255,7 +1397,7 @@ subroutine wrap_json_file_update_real(me,name,val,found)
 
     class(json_file),intent(inout)       :: me
     character(kind=CDK,len=*),intent(in) :: name
-    real(RK),intent(in)                  :: val
+    real(RK32),intent(in)                  :: val
```

Comment:
Some white space should be deleted here for stylistic consistency. I think RK was publicly exposed, but I doubt anyone was using it; the only kind parameter that should have a large impact on client code is the character kind....